pkg/manager/manager.go

```diff
@@ -401,6 +405,100 @@ func New(config *rest.Config, options Options) (Manager, error) {
 	}, nil
 }
 
+// AndFrom will use a supplied type and convert to Options
+// any options already set on Options will be ignored, this is used to allow
+// cli flags to override anything specified in the config file
+func (o Options) AndFrom(loader config.DeferredFileLoader) (Options, error) {
```

Comment:
nit: Do we want to this to accept the interface and do a type check so folks could implement their own configuration providers?

Reply:
It's useful for testing none the less, as I used in `manager_test.go` are you thinking it might be too much customization to allow this?

Reply:
resolved offline.  it is good the way it is.